docs/source/cognite_client.rst

```diff
@@ -1,6 +1,7 @@
 CogniteClient
 =============
 
+.. _class_client_CogniteClient:
```

Comment:
What does this do?

Reply:
I wasn't able to get a link directly to [this](https://cognite-sdk-python.readthedocs-hosted.com/en/latest/cognite_client.html#cognite.client.CogniteClient) part of the docs before, it only let me get to [here](https://cognite-sdk-python.readthedocs-hosted.com/en/latest/cognite_client.html#) by using `:ref:'cognite_client:CogniteClient'` (which is almost the same but not exactly in my opinion but I can switch it to using the second one if preferred)